storm-core/src/jvm/org/apache/storm/utils/Time.java

```diff
@@ -98,10 +104,26 @@ public static long currentTimeMillis() {
             return System.currentTimeMillis();
         }
     }
-    
+
+    public static long secsToMillis (int secs) {
+        return 1000*(long) secs;
+    }
+
+    public static long secsToMillisLong(double secs) {
+        return (long) (1000 * secs);
+    }
+
     public static int currentTimeSecs() {
         return (int) (currentTimeMillis() / 1000);
     }
+
+    public static int deltaSecs(int timeInSeconds) {
+        return Time.currentTimeSecs() - timeInSeconds;
+    }
+
+    public static long deltaMs(long timeInMilliseconds) {
```

Comment:
This one we need a comment on too indicating that this is from System.currentTimeMillis and not Simulated Time.  Not sure if we can use Simulated Time here or not.  If we can it makes things simpler.